lib/utils/colorSpace.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const hueToRgb = (t1, t2, hue) => {
+  if (hue < 0) hue += 6;
+  if (hue >= 6) hue -= 6;
+
+  if (hue < 1) return (t2 - t1) * hue + t1;
+  else if (hue < 3) return t2;
+  else if (hue < 4) return (t2 - t1) * (4 - hue) + t1;
+  else return t1;
+};
+
+// https://www.w3.org/TR/css-color-4/#hsl-to-rgb
```

Comment:
For future PRs, please make sure to use the latest editor's draft, as found on https://drafts.csswg.org/. For this case, it'll be https://drafts.csswg.org/css-color-4/#hsl-to-rgb.